go.mod

```diff
@@ -1,32 +1,36 @@
 module github.com/zlabjp/spire-openstack-plugin
 
-go 1.14
+go 1.16
```

Comment:
It would be nice to update "Prerequisites" in CONTRIBUTING.md depending on this change.
https://github.com/zlabjp/spire-openstack-plugin/blob/b03adc9286448ea087cddf23dc181ec60a20dcf4/CONTRIBUTING.md

Reply:
6fe084d6